styles.css

```diff
@@ -0,0 +1,98 @@
+/* Topnav CSS */
+.topnav {
+  display: flex;
+  width: inherit;
+  height: 72px;
+}
+
+/* Style the links inside the navigation bar */
+.topnav a {
+  font-family: "Inter", sans-serif;
+  float: left;
+  color: #172b4d;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+/* Change the color of links on hover */
+.topnav a:hover {
+  color: #36b37e;
+}
+
+/* Add a color to the active/current link */
+.topnav a.active {
+  color: #36b37e;
+  text-decoration: underline;
+}
+
+/* FLexbox for the header */
+.right {
+  flex-grow: 1;
+}
+
+.left {
+  flex-grow: 3;
+}
+
+.nav-menu {
+  flex-grow: 2;
+}
+
+/* Headline CSS */
+.background-image {
+  display: flex;
+}
+
+.headline {
+  background-image: url('ressources/5243835.jpg');
+  font-family: Inter, Roboto, sans-serif;
+  letter-spacing: 0.3700000047683716px;
+  text-align: center;
+  padding: 160px 10px 160px 10px;
+}
+
+.headline-text1 {
+  color: #172b4d;
+  font-size: 48px;
+  font-style: normal;
+  font-weight: 800;
+  line-height: 60px;
+  letter-spacing: 0.3700000047683716px;
+  text-align: center;
+}
+
+.headline-text2 {
+  color: #36b37e;
+  font-size: 48px;
+  font-style: normal;
+  font-weight: 800;
+  line-height: 60px;
+  letter-spacing: 0.3700000047683716px;
+  text-align: center;
+}
+
+.headline-details {
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 32px;
+  text-align: center;
+  flex: none;
+  margin: 24px 10px;
+}
+
+/* Style all font awesome icons */
+.fa {
+  flex: auto;
+  color: #172b4d;
+  font-size: 18px;
+  padding: 10px;
+  width: 30px;
+}
```

Comment:
- [ ] **[OPTIONAL]** The `flex` property is not necessary here, but as you wrap the buttons into a list, you will need to add the `flex` to the `<ul>` in order for them to stay aligned!

- [ ] **[OPTIONAL]** I can see that there is an underline in your `<a>` tags. You can remove them by simply adding a `text-decoration: none` to the `.fa`.